build/Dockerfile-go1.22.1-solc0.8.13-ubuntu-20.04

```diff
@@ -30,7 +30,7 @@ RUN apt install -yq tzdata && \
     ln -fs /usr/share/zoneinfo/America/New_York /etc/localtime && \
     dpkg-reconfigure -f noninteractive tzdata
 RUN apt install -y ca-certificates libboost-all-dev git make gcc libc-dev curl bash python3 python3-dev python3-pip unzip
-RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && unzip awscliv2.zip && ./aws/install && rm awscliv2.zip
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-aarch64.zip" -o "awscliv2.zip" && unzip awscliv2.zip && ./aws/install && rm awscliv2.zip
```

Comment:
Isn't this file used for both x64 and aarch64? Doesn't it break x64 build this time?